buildscripts/checkstyle.xml

```diff
@@ -121,7 +121,6 @@
             <property name="ignoreFinal" value="false"/>
             <property name="allowedAbbreviationLength" value="1"/>
         </module>
-        <module name="OverloadMethodsDeclarationOrder"/>
```

Comment:
This is checked by errorprone so no need to check in checkstyle (which was annoying since I wanted to suppress in the vendored code)